lbrynet/conf.py

```diff
@@ -193,7 +193,7 @@ def _convert_value(value):
     'pointtrader_server': (str, 'http://127.0.0.1:2424'),
     'reflector_port': (int, 5566),
     'reflector_reupload': (bool, True),
-    'reflector_servers': (list, [('reflector.lbry.io', 5566)], server_port),
+    'reflector_servers': (list, [('jack.lbry.tech', 5566)], server_port),
```

Comment:
I'll revert that before merging, that's just so others can test it before going live on the production reflector.